code/modules/jobs/job_types/gimmick.dm

```diff
@@ -18,6 +18,7 @@
 
 	display_order = JOB_DISPLAY_ORDER_ASSISTANT
 	departments = DEPARTMENT_SERVICE
+	rpg_title = "Peasant"
```

Comment:
Each subtype of gimmick should have an individual title. Our beloved VIP will become peagant, NOT ACCEPTABLE lol